dev_docs/maps/index.md

```diff
@@ -5,5 +5,7 @@ permalink: /dev_docs/maps/
 ---
 
 - [Map Making]({{ "/dev_docs/maps/map_making" | prepend: site.baseurl }})
+- [Map Admin]({{ "/dev_docs/maps/map_admin" | prepend: site.baseurl }})
```

Comment:
The old map_making link should be removed... 
I accidentally overlooked this while reviewing...
Please open another PR for this